dependencies/ruby/Gemfile

```diff
@@ -10,8 +10,10 @@ gem 'openstudio-workflow', '= 1.3.2'
 gem 'openstudio-standards', '= 0.2.0'
 #gem 'openstudio-standards', :github => 'NREL/openstudio-standards', :ref => '1a6a81d242f1783ace78c737059d2ef6121e5675'
 
-gem 'rspec', '= 3.5.0'
-gem 'test-unit-minitest', '= 0.9.1'
-gem 'minitest-reporters', '= 0.14.24'
-gem 'parallel', '= 1.10.0'
+gem 'openstudio_measure_tester', '= 0.1' # This includes the dependencies for running unit tests, coverage, and rubocop
+
+#gem 'rspec', '= 3.5.0' # TODO: is this still needed? Measures use minitest.
+#gem 'test-unit-minitest', '= 0.9.1' # TODO: is this still needed?
+
+gem 'parallel', '= 1.12.1'
 gem 'bundler', '= 1.14.4'
```

Comment:
are we not able to update this to 1.16.1

Reply:
Updating that gem will require updating Ruby (e.g. running the gem update command) on build machines.  It's possible but would prefer not to if we can help it.

Reply:
Okay. I think we just need to change this here and in OpenStudio-measure-tester
https://github.com/NREL/OpenStudio-server/blob/ba1cfa6637886d23f648c19bf8d5178d8ce80a94/docker/deployment/scripts/install_ruby.sh#L8